calico-cloud_versioned_docs/version-3.16/image-assurance/scan-image-registries.mdx

```diff
@@ -246,7 +246,26 @@ To change the CVSS threshold values, note the following:
 
 **Enable Runtime View**
 
-When Runtime View is enabled in Scan Settings, {{prodname}} actively monitors pods running in your Kubernetes clusters so it can correlate them with images you have recorded in the Image Assurance system. By default, this feature is disabled. If you disable the feature after it has been enabled, runtime information that is already collected for existing images will remain in the system.
+You can configure your scan settings to directly monitor pods that are running in your Kubernetes cluster.
+When enabled, the Runtime View service checks image information from the active container images in the cluster.
```

Comment:
"from the active container images in the cluster.", this is for all managed clusters, it's a global setting that is either on or off for all connected managed clusters.

Reply:
Changed.